components/model/src/ome/model/internal/Permissions.java

```diff
@@ -205,10 +205,8 @@ public boolean isSet(Flag flag) {
 
     /**
      * returns the order of the bit representing the given {@link Flag}. This
-     * is dependent on the internal representation of {@link Permission} and
+     * is dependent on the internal representation of {@link Permissions} and
      * should only be used when necessary.
-     *
-     * @see ome.tools.hibernate.SecurityFilter
```

Comment:
Makes sense. Do you have a clear list of the ones you want to keep? Want to create a follow-up item to restore them https://trello.com/c/RpJxX7S7/35-javadoc-cleanup?